config/config.go

```diff
@@ -95,11 +86,6 @@ type (
 	}
 )
 
-// CopySession allows systems to copy the resources session. Necessary for threading.
-func (r *Resources) CopySession() *mgo.Session {
```

Comment:
Do we really need r.CopySession() when we can do r.Session.Copy()? I removed it and fixed up the broken code